### ISSUE TITLE
Godot4 beta 16: Crash when loading a .wav-file

### ISSUE THREAD

**Issue by codespren** [godotengine/godot#72234]
### Godot version

4.0 beta 16 mono

### System information

macOS Ventura 13.1

### Issue description

When loading a .wav-file in beta 16 the game crashes (same project works in beta 15):

```
================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.0.beta16.mono.official (518b9e5801a19229805fe837d7d0cf92920ad413)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] sigsegv_handler(int, __siginfo*, void*)
[2] 2   libsystem_platform.dylib            0x00007ff802102c1d _sigtramp + 29
[3] 3   ???                                 0x0000000000000000 0x0 + 0
[4] JSON::get_data() const
[5] Object* ClassDB::creator<mono_bind::GodotSharp>()
[6] 6   ???                                 0x00000001299f46cf 0x0 + 4993271503
[7] 7   ???                                 0x00000001299f4572 0x0 + 4993271154
[8] 8   ???                                 0x00000001299f1d33 0x0 + 4993260851
[9] 9   ???                                 0x00000001299f15eb 0x0 + 4993258987
[10] 10  ???                                 0x00000001299f024d 0x0 + 4993253965
[11] Object* ClassDB::creator<MobileVRInterface>()
[12] Object* ClassDB::creator<MobileVRInterface>()
[13] Object* ClassDB::creator<mono_bind::GodotSharp>()
[14] 14  ???                                 0x00000001299eff4c 0x0 + 4993253196
[15] 15  ???                                 0x00000001299efe7c 0x0 + 4993252988
[16] 16  ???                                 0x00000001299efb06 0x0 + 4993252102
[17] 17  ???                                 0x00000001299f116d 0x0 + 4993257837
[18] 18  ???                                 0x00000001299eeecb 0x0 + 4993248971
[19] 19  ???                                 0x00000001299eedf9 0x0 + 4993248761
[20] 20  ???                                 0x00000001299eed7c 0x0 + 4993248636
[21] 21  ???                                 0x00000001299e7516 0x0 + 4993217814
[22] 22  ???                                 0x00000001299e698c 0x0 + 4993214860
[23] 23  ???                                 0x00000001299e66b4 0x0 + 4993214132
[24] 24  ???                                 0x00000001299e635d 0x0 + 4993213277
[25] Object* ClassDB::creator<MobileVRInterface>()
[26] MultiplayerAPI::is_server()
[27] Node3D::set_ignore_transform_notification(bool)
[28] ScriptInstanceExtension::_add_property_with_state(void const*, void const*, void*)
[29] Node::get_tree() const
[30] Node::get_tree() const
[31] Node::is_inside_tree() const
[32] Node::is_inside_tree() const
[33] 33  Godot                               0x000000010caf666d Godot + 4642413
[34] RendererCompositorRD::_create_current()
[35] 35  dyld                                0x00007ff801da5310 start + 2432
-- END OF BACKTRACE --
================================================================

```
### Steps to reproduce

1. Load a .wav-file from the code: `AudioStreamWav audioStream = ResourceLoader.Load<AudioStreamWav>(path);`
2. Crash

### Minimal reproduction project

[godot4beta16test.zip](https://github.com/godotengine/godot/files/10526732/godot4beta16test.zip)

**Comment by paulloz** [comment]
Should be fixed by #72205

**Comment by akien-mga** [comment]
Duplicate of https://github.com/godotengine/godot/issues/72199 (same cause).
